readme.txt

```diff
@@ -116,6 +116,22 @@ You'll find the [FAQ on Yoast.com](https://yoast.com/wordpress/plugins/seo/faq/)
 
 == Changelog ==
 
+= 3.9.0 =
+
+Release Date: November 29nd, 2016
+
+* Enhancements:
+    * Updates the banners on the admin pages.
+    * Improves accessibility by moving the Google Search Console reload button from the header.
+    * Allow for other plugins and themes to more easily add html namespaces through the new wpseo_html_namespaces filter.
+    * Prevent conflicts with other plugins/themes which also add html namespaces.
+
+* Bugfixes:
+    * Adds a check to prevent a "Cannot read property 'body' of undefined" error with tinyMCE that occurred with Visual Composer and some themes.
```

Comment:
Some themes is indeed not very specific, however due to the nature of the problem it is very difficult to say if the tinyMCE error was caused by a theme or plugin. So pinpointing if a theme or plugin was causing the error is very difficult to say.